apollo-client/src/main/java/io/logz/apollo/clients/ApolloClient.java

```diff
@@ -249,6 +249,10 @@ public Group updateGroup(int groupId, String name, int serviceId, int environmen
         });
     }
 
+    public void deleteGroup(int id) throws IOException {
```

Comment:
Some methods are throwing **IOException** and some **ApolloClientException**.
shouldn't we throw the latter for consistency?

Reply:
delete method cannot throw **ApolloClientException** (compilation error)